[16.0][IMP] sale_timesheet_line_exclude: add security group

sbidoul opened the pull request:
closes #705 

This should be merged with a minor or major version bump as it adds a security group that will need to be added to relevant users in order to preserve functionality.

sbidoul committed:
[IMP] sale_timesheet_line_exclude: add security group

alexey-pelykh commented on sale_timesheet_line_exclude/security/exclude_from_sale_order.xml:
```diff
@@ -0,0 +1,12 @@
+        <field name="category_id" ref="base.module_category_hidden" />
+        <field name="comment">Gives access to the Non-Billable field on timesheets.</field>
+    </record>
+</odoo>
```
Formatter does not add a new-line?

sbidoul replied:
It does as soon as I run it :)

alexey-pelykh reviewed (changes requested):
It's worth mentioning this in the README.

alexey-pelykh commented:
I feel like you didn't add the file you wanted to add :)

sbidoul commented:
> I feel like you didn't add the file you wanted to add :)

What do you mean?

sbidoul commented:
Ah, the README, updated now.

sbidoul committed:
[FIX] sale_timesheet_line_exclude: don't copy the field

We don't copy the field by default, because it breaks the
OE timesheet grid for users in the security group that hides
the field.

Functionally, it also makes sens to not copy this field,
because it is better to consider the new line not excluded
by default.

sbidoul commented:
I discovered a bug where the security group breaks the timesheet grid, which attemps to copy timesheet lines.

So I declare the field copy=False, which makes sense since we normally want the copied lines to be "not excluded" by default.

sbidoul commented:
/ocabot merge minor
